src/modules/comments/renderComments.js

```diff
@@ -0,0 +1,80 @@
+import postComments from './setComments';
+import getComments from './getComments';
+import commentCounter from './commentCounter';
+
+const renderFoodComments = (commets) => {
+  const ulComments = document.getElementById('comments');
+  ulComments.innerHTML = '';
+  commets.forEach((element) => {
+    ulComments.innerHTML += `<li>${element.creation_date} ${element.username}: 
+    ${element.comment}</li>`;
+  });
+  commentCounter();
+};
+
+const loadComments = (itemId) => {
+  getComments(itemId).then((data) => {
+    renderFoodComments(data);
+  }).catch(() => { commentCounter(); });
+};
+
+const renderComments = (data, itemId) => {
+  const renderLocation = document.getElementById('food-container');
+  renderLocation.classList.remove('disable');
+  renderLocation.innerHTML = `
+  <div class="food">
+  <span id="close-comment">X</i></span>
+    <div class="food-img">
+      <img class="image-popup" src=${data.meals[0].strMealThumb} alt="food">
+      <h2 class="title">${data.meals[0].strMeal}</h2>
+    </div>
+      <ul class="food-info">
+        <li>
+          <ul>
+          <li>Category: ${data.meals[0].strMeal}</li>
+            <li>Category: ${data.meals[0].strCategory}</li>
+          </ul>
+        </li>
+        <li>
+          <ul>
+          <li>Area: ${data.meals[0].strArea}</li>
+            <li><a href=${data.meals[0].strYoutube}>How To Make</a></li>
+          </ul>
+        </li>
+      </ul>
+    <div class="food-comments">
+      <div class="comments-container">
```

Comment:
- [ ] Kindly ensure that you improve the size of the design of the `Comment Popup` in order to looks smarter and sticks as much as possible to the given **[wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/group-capstone/images/Reservations.png)**(e.g.: **Espacially the size of the popup,...**)

Reply:
Added changes requested